app/models/car_park.rb

```diff
@@ -0,0 +1,6 @@
+class CarPark < ApplicationRecord
+  belongs_to :owner, class_name: 'User', foreign_key: :user_id
+
```

Comment:
```suggestion
  belongs_to :user
```
Зачем городить лишнее
Уж если хочешь вызывать на паркинге овнера
```suggestion
  belongs_to :owner, class_name: 'User', foreign_key: :user_id
  
  def owner
    user
  end
  
  или
  
  alias owner user
```